<launch>
  <arg name="filter_config" default="$(find dbrt)/config/rbc_particle_filter_tracker.yaml"/>
  <arg name="camera_config" default="$(find dbrt)/config/camera.yaml"/>
  <arg name="camera" default="XTION" />

  <param name="use_sim_time" type="bool" value="false" />
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find arm_robot_model)/models/DarpaRobot.urdf.xacro'" />

  <!-- Pose Estimation Node-->
  <node name="rbc_particle_filter_robot_tracker_simulation" type="rbc_particle_filter_robot_tracker_simulation" pkg="dbrt" output="screen" respawn="false" launch-prefix="">
    <param name="robot_description_package_path" type="string" value="$(find arm_robot_model)" />
    <param name="camera_frame"  type="string"  value="$(arg camera)" />

    <rosparam command="load" file="$(arg filter_config)" />
    <rosparam command="load" file="$(arg camera_config)" />
  </node>
</launch>
